### PR TITLE
Use of Funds Campaign Configuration

### PR DESCRIPTION
**Pull request by Abban**
Move some of the use of funds data into campaign configured items

Ticket: https://phabricator.wikimedia.org/T321653

### CODE REVIEW

File: components/UseOfFunds/FundsContent.jsx [hunk 1]
```diff
@@ -68,6 +68,7 @@ export default function FundsContent( props ) {
 		<div className="banner_model__section use_of_funds__section--call_to_action">
 			<button className="use_of_funds__button" onClick={onCallToAction}>{content.callToAction}</button>
 		</div>
+		{ props.figuresAreProvisional && ( <div style="text-align: left; font-size: small; padding-bottom: 16px;">*) vorläufig</div> ) }
```
Comment (gbirke) [inline components/UseOfFunds/FundsContent.jsx:71]: This needs to be translatable for the EN banners

File: shared/override_use_of_funds_figures.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+export default function overrideUseOfFundsFigures( useOfFundsMessages, campaignConfigurationOverrides ) {
+	for ( const key in campaignConfigurationOverrides ) {
+		const item = useOfFundsMessages.applicationOfFundsData.find( x => x.id === key );
+		if ( item ) {
+			item.percentage = campaignConfigurationOverrides[ key ];
+		}
+	}
+	return useOfFundsMessages;
+}
```
Comment (gbirke) [inline shared/override_use_of_funds_figures.js:9]: I like this function - short, to the point, readable, catches relevant errors and allows for maximum extensibility, both in use of funds and overrides. :+1:

**Review by gbirke — COMMENTED** [review]
This is still incomplete, right?

File: components/UseOfFunds/FundsContent.jsx (hunk 1 above)
Comment (Abban) [reply components/UseOfFunds/FundsContent.jsx:71]: Whoops! It is translatable already, copy/paste error..

**Comment by Abban** [comment]
>This is still incomplete, right?

Sorry just saw this, it's completed and integrated into all the banners

**Review by gbirke — APPROVED** [review]
Looks good, the EN desktop banner still has the untranslated string

**Comment by Abban** [comment]
>Looks good, the EN desktop banner still has the untranslated string

Oh no, I forgot to merge the changes I guess. I'll redeploy it.

**Pull request merged**